### PR TITLE
Bylaw review / proposed edits

### PR DESCRIPTION
**Pull request by ebullient**
Keeping Trademark policy separate, see #70

### CODE REVIEW

File: bylaws/5-decision-making.md [hunk 1]
```diff
@@ -46,7 +46,7 @@ ensuring a comprehensive understanding of the community's stance.
 
 Discussion of the proposal should occur in comments on the PR or GitHub Discussion. If conversations happen elsewhere (in chat, for example), a summary of the conversation should be added as a comment if the discussion helped clarify the proposal or resolve a concern.
 
-If significant changes are needed, the current Discussion/PR is closed as outdated, and a new, clearer proposal is initiated.
+If significant changes are needed, the current Discussion/PR should closed as outdated, and replaced with a revised proposal.
```
Comment (kenfinnigan) [inline bylaws/5-decision-making.md:49]:
```suggestion
If significant changes are needed, the current Discussion/PR should be closed as outdated, and replaced with a revised proposal.
```

**Review by kenfinnigan — APPROVED** [review]
Minor nit

**Review by kenfinnigan — APPROVED** [review]

**Comment by nmcl** [comment]
Looks good to me.

**Comment by ebullient** [comment]
Merging as benchmark to move forward from.

**Pull request merged**